### PR TITLE
Adding tripactions tile

### PR DESCRIPTION
**Pull request by limed**

### CODE REVIEW

**Comment by limed** [comment]
Requesting bug is https://bugzilla.mozilla.org/show_bug.cgi?id=1672012

This adds the tripactions tile to the sso dashboard

File: apps.yml [hunk 1]
```diff
@@ -4030,7 +4030,7 @@ apps:
     authorized_users: []
     client_id: i1qBrMjJEdKlTNEVgGwc9P0xFxI9cuD8
     display: false
```
Comment (The-smooth-operator) [inline apps.yml:4032]: You have to set `display: true` for the tile to be displayed

**Review by The-smooth-operator — CHANGES_REQUESTED** [review]
Thanks for doing this Ed!

File: apps.yml (hunk 1 above)
Comment (limed) [reply apps.yml]: Fixed in ed01bf844006be2bf409692382fea31d774018ad

**Review by The-smooth-operator — APPROVED** [review]
looks good now, thanks!

**Pull request merged**